db/migrate/20220326155350_create_likes.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
```

Comment:
- [ ] Let's add the code so it will create author_id and a corresponding index and  will create post_id and a corresponding index. 
```ruby
t.references :author, foreign_key: { to_table: 'users' } 
t.references :post, foreign_key: true
 
```

Reply:
Okay thanks, I will implement them all now 
but for this part of code should I do it manually or there is some migration I can do it with?

Reply:
@arslanbisharat